ceph-iscsi-config.spec

```diff
@@ -1,11 +1,11 @@
 Name:           ceph-iscsi-config
-Version:        2.4
```

Comment:
You should rebase the patch so it is against the current code. The current version is 2.6 so this patch reports merge conflicts.

Reply:
@mikechristie thanks for the reminder. Rebased!